ElevenLabs/Packages/com.rest.elevenlabs/Runtime/Models/TimestampedTranscriptCharacter.cs

```diff
@@ -0,0 +1,33 @@
+using System;
+
+namespace ElevenLabs.Models
+{
+    /// <summary>
+    /// Represents timing information for a single character in the transcript
+    /// </summary>
+    [Serializable]
+    public class TimestampedTranscriptCharacter
+    {
+        /// <summary>
+        /// The character being spoken
+        /// </summary>
+        public readonly string Character;
+
+        /// <summary>
+        /// The time in seconds when this character starts being spoken
+        /// </summary>
+        public readonly double StartTime;
+
+        /// <summary>
+        /// The time in seconds when this character finishes being spoken
+        /// </summary>
+        public readonly double EndTime;
```

Comment:
same thing here since the class is serializable, the fields should be private, not readonly, and have public property accessors